compute/elastic-metal/how-to/use-commitment-fee-payback.mdx

```diff
@@ -0,0 +1,48 @@
+---
+meta:
+  title: How to use the commitment fee payback
```

Comment:
maybe "How to use the commitment fee payback feature" or "How to get a commitment fee payback credit" ?

Reply:
```suggestion
  title: How to use the commitment fee payback feature
```